src/bot/lib/command.ts

```diff
@@ -4,33 +4,42 @@ import {
   CommandInteraction,
   InteractionResponse,
   MessageApplicationCommandData,
-  MessageContextMenuCommandInteraction
+  MessageContextMenuCommandInteraction,
+  UserApplicationCommandData
 } from 'discord.js';
 
 export default class Command {
   public name: string;
 
-  private raw: ChatInputApplicationCommandData | MessageApplicationCommandData;
+  private raw: ChatInputApplicationCommandData | MessageApplicationCommandData | UserApplicationCommandData;
```

Comment:
Étant donné qu'on utilise les 3, je propose de prendre le type de groupement `ApplicationCommandData` :
![image](https://user-images.githubusercontent.com/29425008/175193782-1923127b-0529-4680-a213-32dc9677afac.png)

PS: Le faire partout où on appelle les 3

```suggestion
  private raw: ApplicationCommandData;
```